includes/admin/donors/donor-actions.php

```diff
@@ -138,6 +140,20 @@ function give_edit_donor( $args ) {
 
 	if ( $donor->update( $donor_data ) ) {
 
+		// Set Donor Meta and User Meta for donor first name.
+		if( ! empty( $donor_info['first_name'] ) ) {
+			Give()->donor_meta->update_meta( $donor_id, '_give_donor_first_name', $donor_info['first_name'] );
+
+			// Update User Meta, if user is attached with donor
+			if ( ! empty( $donor->user_id ) ) {
+				update_user_meta( $donor->user_id, 'first_name', $donor_info['first_name'] );
+			}
+		}
+
+		// Set Donor Meta and User Meta for donor last name.
+		Give()->donor_meta->update_meta( $donor_id, '_give_donor_last_name', $donor_info['last_name'] );
```

Comment:
@mehul0810 We have conditional check for `first_name` and `user_id`but not for `last_name` and `user_id`, any specific reason for that?